auth/store_test.go

```diff
@@ -115,6 +119,7 @@ func TestCheckPassword(t *testing.T) {
 	if err != ErrAuthFailed {
 		t.Fatalf("expected %v, got %v", ErrAuthFailed, err)
 	}
+	disableAuth(as)
```

Comment:
put `defer as.AuthDisable()` after `as := NewAuthStore(b)` instead, otherwise it won't clean up if there's a Fatal